source/_docs/installation/troubleshooting.markdown

```diff
@@ -43,3 +43,21 @@ For `iptables` systems (was the default for older distributions):
 iptables -I INPUT -p tcp --dport 8123 -j ACCEPT
 iptables-save > /etc/network/iptables.rules  # your rules may be saved elsewhere
 ```
+
+### System freezes
+
+Upon the first run or after an upgrade, Home Assistant uses a lot of resources to (re)compile all the integrations. 
```

Comment:
I think we should make very clear this is for a specific installation type.

Reply:
I thought it was pretty general, any system can run out of memory. Do you think it specifically only applies to a Raspberry Pi 2?

Reply:
No, it is not generic, as we pre-build and pre-compile all dependencies (Python wheels) for all supported architectures and most installation types.

Reply:
I see your point. Added a line.